src/protocols/patches/ranking_legacy.js

```diff
@@ -0,0 +1,164 @@
+const Packet = require('../../packet'); // eslint-disable-line no-unused-vars
+const PacketV0 = require('../../packetv0'); // eslint-disable-line no-unused-vars
+const PacketV1 = require('../../packetv1'); // eslint-disable-line no-unused-vars
+const RMCMessage = require('../../rmc'); // eslint-disable-line no-unused-vars
+const Stream = require('../../stream');
+
+const Requests = require('../requests/ranking_legacy');
+const Responses = require('../responses/ranking_legacy');
+
+class RankingLegacy {
```

Comment:
A note here about the protocol being rewritten in newer versions of the libraries should be added here for clarity purposes